monkestation/code/modules/job_xp/milestones/_milestones.dm

```diff
@@ -94,3 +97,22 @@
 	if(ishuman(mob))
 		var/mob/living/carbon/human/human_mob = mob
 		human_mob.put_in_hands(new_item)
+
+/client/verb/reload_job_rewards()
+	set category = "IC"
+	set name = "Reload Job Rewards"
+	set desc = "Refresh job rewards you have for the current job (Use this if you suspect something is missing or broken)."
+
+	if(!isliving(mob))
+		to_chat(src, "For this to work you need to be living.")
+		return
+
+	var/datum/mind/listed_mob = mob.mind
+	var/job_string = listed_mob.assigned_role.title
+
+	for(var/datum/job_milestone/subtype as anything in subtypesof(/datum/job_milestone))
+		if(!initial(subtype.key_id) == job_string)
+			continue
+		var/datum/job_milestone/subtype_created = new subtype
+		for(var/level in 1 to mob.client.prefs.job_level_list[job_string])
```

Comment:
checj if job_level_list[job_string] is outputting a number with a text2num check or something getting runtimes about back numbers

Reply:
Right, I think the problem is people who still haven't leveled up in that job. They don't have a 0 as their level, they have nothing, so I'll add a check for that.